SPECS/Cython/Cython.spec

```
@@ -1,11 +1,13 @@
 %global upname cython
+%global _description \
+Cython is an optimising static compiler for both the Python programming language and the extended Cython programming language (baded on Pyrex). It makes writing C extensions for Python as easy as Python itself.}
+Summary:        Language for writing Python extension modules
 Name:           Cython
-Version:        0.29.26
+Version:        0.29.32
 Release:        1%{?dist}
-Summary:        Language for writing Python extension modules
-Vendor:         Microsoft
-Distribution:   Mariner
 License:        ASL 2.0
```

License reference: https://spdx.org/licenses/Apache-2.0.html

Since we're moving to SPDX short identifiers for licenses, mind making the following change?

```suggestion
License:        Apache-2.0
```